lib/zeebe/assignment_definition.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Zeebe
+  class AssignmentDefinition
+    attr_accessor :assignee, :candidate_groups, :candidate_users
```

Comment:
Given all of these have identical implementations, it might be worth something like:

```ruby
moddle_acessor :assignee, :candidate_groups, :candidate_users

def self.moddle_accessor(*attributes)
  attributes.each do |attribute|
    define_method attribute do
      moddle[:attribute.to_s.camelize(:lower)]
    end
  end
end
```

Note that this method is read-only and eschews the use of instance variables.

Reply:
After discussion, we intentionally don't want to store the `moddle`, which may be huge and annoying to have in every object.

These should be readers though.